### ISSUE TITLE
&dungeon needs rebalancing and fixing

### ISSUE THREAD

**Issue by Felix-44** [Felix-44/Xilef#13]
the game starts too easily and then becomes too impossible, a rebalance would be great
currently thinking of lowering the cap, but the enemies might need new statistics

**Comment by Felix-44** [comment]
For some reason the Ice attack doesnt lower mana but defense instead..there might be other issues too

**Comment by Felix-44** [comment]
I fixed the Ice attack, for the rebalance i'll wait for Dark Brick

**Comment by DarkBrick** [comment]
ill work on it tommorow bcus trip back home will take 2 days

**Comment by Felix-44** [comment]
just realized we should assign @Lemon-Cannon to this since he helped

**Comment by unholy-codeism** [comment]
any new updates btw? this has been open since august ngl

**Comment by Felix-44** [comment]
nope, I've been too busy to work on this and we haven't heard anything from @Lemon-Cannon

**Comment by Felix-44** [comment]
why is this closed??
